### ISSUE TITLE
UpdateVM ignores updates-proxy-setup feature

### ISSUE THREAD

**Issue by DemiMarie** [QubesOS/qubes-issues#6274]
**Qubes OS version**

`Qubes release 4.0 (R4.0)`

**Affected component(s) or functionality**

`qubes-dom0-update`

**Brief summary**

If I use `qvm-features` to tell the UpdateVM to use the updates proxy, it is ignored.

**How Reproducible**

100%

**To Reproduce**

1. Create a named `sys-update` with no NetVM, and set it as UpdateVM.
2. Run `qvm-features sys-update updates-proxy-setup 1`
3. Run `qvm-tags add sys-update whonix-updatevm`, so that `sys-update` can update via `sys-whonix`.
3. Run `qubes-dom0-update`

**Expected behavior**

Updates succeed

**Actual behavior**

Updates fail, as the updates proxy is not used.

**Screenshots**

**Additional context**

**Solutions you've tried**

Manually editing `/etc/yum.conf` in dom0 works.

**Relevant [documentation](https://www.qubes-os.org/doc/) you've consulted**

**Related, [non-duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) issues**

**Comment by marmarek** [comment]
That actually sounds more like a feature request. [Updates proxy](https://www.qubes-os.org/doc/software-update-domu/#updates-proxy) is a mechanism designed for domU updates download, not dom0 - to keep a template isolated from other network access. For dom0 updates, UpdateVM already fills that role, so there is very little (if at all) sense to use extra proxy in between. This is especially the case if you create dedicated VM for this role, instead of default sys-firewall.

**Comment by DemiMarie** [comment]
Good point @marmarek.  In my case, the reason is that my UpdateVM is based on a minimal template with no networking support. That is indeed an edge case, however, and the workaround is simple.
